rust/lance-linalg/build.rs

```diff
@@ -58,7 +69,9 @@ fn main() {
         // While GCC doesn't have support for _Float16 until GCC 12, clang
         // has support for __fp16 going back to at least clang 6.
         // We use haswell since it's the oldest CPUs on AWS.
-        build_f16_with_flags("avx2", &["-march=haswell"]).unwrap();
+        if let Err(_) = build_f16_with_flags("avx2", &["-march=haswell"]) {
+            println!("cargo:warning=Unable to build any fp16 kernels.  Please use Clang >= 6 or GCC >= 12");
```

Comment:
Could we not swallow the error here? It's entirely possible there is some other error.

Reply:
Fixed.  The error is no longer swallowed.